docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
+
+**containers-storage:**_docker-reference_
```

Comment:
If only…

[`[`_complex options_`]`] (_docker reference with a tag or/**and** a digest_ | _hex digest prefix_) [`@`_hex digest prefix_]

where “prefix” includes the case of a full digest value.

Reply:
Not a “digest prefix”, but an “image ID (prefix)”. And the above seems to allow _image ID prefix_`@`_image ID prefix`, which is forbidden.

(In hindsight, we should have insisted on a parser generator before starting this textual syntax business…)

Reply:
So,

[`[`_complex options_`]`] ( _image ID prefix_  | _docker reference with a tag or/**and** a digest_ [`@`_image ID prefix_] )

I think.

Reply:
> [[complex options]] ( image ID prefix | docker reference with a tag or/and a digest [@image ID prefix] )
> I think.

This looks good :+1: